server/lib/manifest.py

```diff
@@ -182,10 +195,11 @@ def create_context(self):
         """
         return {
             "@context": [
+                "https://w3id.org/bundle/context",
                 {"schema": "http://schema.org/"},
-                {"DataCite": "http://datacite.org/schema/kernel-4"},
-                {"Datasets": {"@type": "@id"}},
-                "https://w3id.org/bundle/context"
+                {"dc": "http://datacite.org/schema/kernel-4"},
```

Comment:
```suggestion
                {"datacite": "http://datacite.org/schema/kernel-4"},
```
I realize this is more annoying than anything, but because "dc" is widely used for "Dublin Core" I think Tommy and I aggreed that we should go with "datacite".